packages/storefront/src/components/ComponentMetaOverview.vue

```diff
@@ -37,7 +37,7 @@
       return ['', ...tagNames]
         .map((tagName) => {
           const { isDeprecated, isExperimental } = getComponentMeta(tagName as TagName) || {};
-          return `<p-table-head-cell>${tagName}${isDeprecated ? ' 🚫' : ''}${
+          return `<p-table-head-cell ${!tagName && 'aria-hidden="true"'}>${tagName}${isDeprecated ? ' 🚫' : ''}${
```

Comment:
If this evaluates to false it will be applied as attribute:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/1d8cf7fb-3787-49ac-ac2d-d9004e28805a">